Documentation/operation docs rework (#3074)
scality/metalk8s: docs/operation/cluster_monitoring/monitoring_stack.rst

Monitoring Stack
================

> MetalK8s ships with a monitoring stack that provides a cluster-wide view of

Reviewer: Metalk8s is shipped with

Reviewer 2: I'd let it go.

cluster health, pod status, node status, and network traffic status.

> These view points are represented as charts, counts and graphs. For a

Reviewer 2: I'd ditch the paragraph _and_ sentence breaks: 

cluster health, pod status, node status, and network traffic status, using charts, counts, and graphs.

or 

MetalK8s ships with a monitoring stack that uses charts, counts, and graphs to provide a cluster-wide view of cluster health, pod status, node status, and network traffic status.

> closer look, access the :ref:`Grafana Service<installation-services-grafana>`

Reviewer 2: ```suggestion
Access the :ref:`Grafana Service<installation-services-grafana>`
```

> to get insights on monitoring stats provided once MetalK8s is deployed.

Reviewer 2: ```suggestion
for monitored statistics provided once MetalK8s has been deployed.
```

> The MetalK8s monitoring stack is composed of the following main components:

Reviewer 2: ```suggestion
The MetalK8s monitoring stack consists of the following main components:
```

Reviewer 2: you could also say "comprises the following"

  - :term:`Alertmanager`
  - :term:`Grafana`
  - :term:`Kube-state-metrics`
  - :term:`Prometheus`
  - :term:`Prometheus Node-exporter`

.. todo::

   - For each of the components listed above, provide a detailed description of

>      its role within the Monitoring stack.

Reviewer 2: ```suggestion
     its role within the monitoring stack.
```

>    - Default alerting & recording rules are available as a JSON file, we should

Reviewer 2: ```suggestion
   - Default alerting & recording rules are available as a JSON file. We should
```
